templates/codepipeline-cf-template-codecommit-source.yml

```diff
@@ -238,7 +238,6 @@ Resources:
             # Also not sure about this one
             Resource: 
               - !Join [ '', [ 'arn:aws:iam::', !Ref 'AWS::AccountId', ':role/', !Ref Service, '-', !Ref Environment, '-', !Ref 'AWS::Region', '*' ] ]
-              - !GetAtt CBRole.Arn
```

Comment:
I feel like we need to do either some manual testing (trial - error) here, or ask clarification from Gene.

In the bug, he mentions this as a _permission_ problem, but here we are removing the _resource_ instead (which could be a valid way to fix, I just don't know if this will break deployment).

Maybe we need to keep `!GetAtt CBRole.Arn` as a resource, but delete most of the permissions above (and only leave `iam:GetRole` as he suggests).

In the same vein, what about the other resource (the line above `!GetAtt CBRole.Arn`)? Maybe that's safe to leave as-is as he did not refer to it.

I will do some testing, and if I can't figure this out I will respond to the bug.

Reply:
I've already tested this, it works

Reply:
Awesome, thanks for testing this without it. I did not have the time.

I am still curious though if we need all those permissions for that resource (this is probably nitpicking).